### DISCUSSION TITLE
FCB1010 and Grandorgue

### DISCUSSION THREAD

**Discussion by larspalo** [GrandOrgue/grandorgue#515] [Help]
Originally posted 2018-05-19 13:16:06.346000 by jer in GrandOrgue discussion forum on SourceForge<br><br>Could someone please help me with setting up the FCB1010 with Grandorgue  V. 0.3.1.2291. I am mainly interested in getting the two Volume pedals to communicate with GO.
Thanks in advance.
Jerry

**Comment by larspalo** [comment]
Originally posted 2018-05-20 23:38:08.762000 by [Martin Koegler](https://github.com/e9925248) in GrandOrgue discussion forum on SourceForge<br><br>sorry-wrong thread

**Comment by larspalo** [comment]
Originally posted 2018-05-20 23:39:48.619000 by [Martin Koegler](https://github.com/e9925248) in GrandOrgue discussion forum on SourceForge<br><br>Did you try the &quot;detect complex MIDI event&quot; function?

**Comment by dcp9** [comment]
My experience is with a Mac Mini M4 running current GrandOrgue. Maybe a Linux or Windows person has other advice.

1. Restore all defaults to the FCB1010 by holding down UP and DN while powering on.
2. Attach a usb to midi cable between computer and FCB1010
3. Open Audio MIDI Setup on Mac, open the MIDI studio window. The MIDI Cable will show up. Right Click to get the properties sheet for the new device and name it something useful, like FCB1010.
4. Start GrandOrgue. Open MIDI and Audio Settings, open the MIDI Devices panel. Turn on the FCB1010 as an input.
5. In the Organ’s window (or the coupler and volume window) Right click on the expression pedal you want to control. In the dialog box click Listen for Event. Open or close the pedal on the FCB1010 and the dialog should show the pedal has sent an event.
6. Click OK.
7. Check that the expression pedal is now controlling the Organ as desired.

Troubleshooting:
1 Try a reboot of the computer and power the FCB1010 off then on.
2 There are free apps that you can run on the computer to see what MIDI messages are being received and sent from devices. These can validate the functioning of the hardware and the computer communications.
3 You might need to change MIDI channels for the FCB1010.
